tutorialdb/settings.py

```diff
@@ -119,6 +119,7 @@
 USE_TZ = True
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.environ['STATIC_ROOT']
 
 STATICFILES_DIRS = (os.path.join('static'), )
```

Comment:
**`STATIC_ROOT`** and **`STATICFILES_DIRS`** can't co-exist in the settings.

We will be updating the **`settings.py`** file, so don't change anything here.

Reply:
Ok, I'll remove it for now

Reply:
I guess I was mistaken. But I couldn't run **`collectstatic`** locally with both of them set.

Reply:
That's strange. 
Need to double check